sdk/formrecognizer/Azure.AI.FormRecognizer/src/CustomLabeledModel.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Azure.AI.FormRecognizer.Models;
+
+namespace Azure.AI.FormRecognizer.Custom
+{
+    public class CustomLabeledModel
+    {
+        internal CustomLabeledModel(Model_internal model)
+        {
+            ModelId = model.ModelInfo.ModelId.ToString();
+            AverageLabelAccuracy = model.TrainResult.AverageModelAccuracy.Value;
+            LabelAccuracies = ConvertLabelAccuracies(model.TrainResult.Fields);
+            TrainingStatus = new CustomModelTrainingStatus(model.ModelInfo);
+            TrainingInfo = new TrainingInfo(model.TrainResult);
+        }
+
+        public string ModelId { get; internal set; }
```

Comment:
You might not need all the internal setters if you only set properties from the ctor.